contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -5550,7 +5550,12 @@ makeFetchCursorStatement(TSqlParser::Fetch_cursorContext *ctx)
 	/* cursor_name */
 	auto targetText = ::getFullText(ctx->cursor_name());
 	result->curvar = lookup_cursor_variable(targetText.c_str())->dno;
+    
+	/* FETCH CURSOR without destination should be blocked inside a function. */
 
+	if (is_compiling_create_function() && !ctx->INTO()){
```

Comment:
indentation

Reply:
Indentation seems correct.

Reply:
Curly braces in next line
if (conditions)
{
}

Reply:
updated